app/bundles/InstallBundle/Install/InstallService.php

```diff
@@ -308,19 +308,20 @@ public function createDatabaseStep($step, $dbParams)
     {
         $translator = $this->translator;
 
-        $messages = $this->validateDatabaseParams($dbParams);
+        $status   = $this->validateDatabaseParams($dbParams);
```

Comment:
validateDatabaseParams() is returning an array of validation errors as well which would be ignored with this change.